readme.md

```diff
@@ -34,6 +34,7 @@ __Build what matters__
 __Do *big* things in small steps__
   
   * Breaking down big features in small steps doesn’t mean we work only on small incremental improvement. A longer term vision or objective help us achieve great things without sacrificing short term value.
+  * Make use of feature flags to allow you release early and often to test assumptions, especially in production.
```

Comment:
Looking at previous examples, I think it should be `allow us` rather than `you`

Reply:
Is 'especially' redundant? We use feature flags so that we can test in production

Reply:
We also use them in test suites. Tests run in UK/DE/ES environments, i.e. with all relevant features enabled/disabled.

Reply:
I think I meant more `test against production-scale data safely`, since LaunchDarkly lets you rollout partially.